### DISCUSSION TITLE
"npm i author/repo" works, but "npm remove author/repo" doesn't

### DISCUSSION THREAD

**Discussion by ftzi** [npm/feedback#235] [CLI] [answered]
After making some changes to my package repository wip branch, I found out to install it via `npm i githubUsername/reponame#wip`. Ok! I decided to remove it after some time, and logically just changed the `i` to `remove`. It just doesn't work, nothing happens. I had to normally `npm remove packageName` to do it.

Not a big deal, but I think it should be implemented.

**Comment by MylesBorins** [comment, marked as answer]
Thanks @SrBrahma 

If you would like to suggest this as a feature for the npm cli you should open an issue over at https://github.com/npm/rfcs

Thanks for stopping by 🎉
